### ISSUE TITLE
Search page continuously refresh

### ISSUE THREAD

**Issue by andreesg** [plone/Products.CMFPlone#1631]
Noticed the issue mentioned by tkimnguyen in https://github.com/plone/ploneorg.core/issues/160 happening often only while using Safari.

How to reproduce:
* Plone 5.0.4
* Safari 8.0.8
* Visit https://plone5.veit-schiele.de/search or http://plone.org/search

The issue doesn't happen with the latest Safari 9

**Comment by tkimnguyen** [comment]
Yeah I just tried on plone.org and it's not happening for me either (Safari 9.1.1 on OS X 10.11.5

**Comment by jensens** [comment]
Closing as outdated (10 years old). Specific to Safari 8.0.8 on Plone 5.0.4. Safari 8 is over a decade old and the search page JavaScript has been completely rewritten.
If you think this issue is still relevant please reopen it. Thank you for the original report!
